packages/backend/src/services/HealthService/HealthService.ts

```diff
@@ -108,4 +91,19 @@ export class HealthService {
                 this.lightdashConfig.headlessBrowser?.host !== undefined,
         };
     }
+
+    private hasSlackConfig(): boolean {
```

Comment:
This method was also updated on https://github.com/lightdash/lightdash/pull/7376